docs/source/resources/define-metadata.rst

```diff
@@ -1299,9 +1299,10 @@ You can specify files to be included in the package in 1 of
 Explicit file lists are relative paths from the root of the
 build prefix. Explicit file lists support glob expressions.
 Directory names are also supported, and they recursively include
-contents.
+contents. Files installed to the prefix by host dependencies will
+be matched by glob expressions.
```

Comment:
I would maybe even highlight this further with an admonition or something similar. I wasn't aware of that design issue!

Reply:
> with an admonition or something similar.

I don't know what you mean. You want stronger language? For example: "WARNING! Files installed to the prefix by host dependencies also will be matched by glob expressions."

> I wasn't aware of that design issue!

Yeah, I'm not sure whether this is a bug/oversight or a feature. It's definitely unexpected to me. The most active contributor in this section is @msarahan; maybe they know?

Reply:
I meant an [admonition in the RST sense](https://documatt.com/restructuredtext-reference/admonitions.html). Triple-backtick + `warning` kind of syntax. Equivalent to this:

> [!WARNING]
> Files installed to the prefix by host dependencies also will be matched by glob expressions.

Reply:
Done in 642906b404b72aa3dc6b2d3342e295520585d65f